tests/mappings/test_mappings.py

```diff
@@ -51,8 +51,9 @@ def test_pv_load_configuration():
     pv_configuration = PVLoadConfiguration.load_default_pv_configuration(
         suffix="_test",
         granted_files=random.sample(AVAILABLE_MAPPING_FILES["granted"], k=5),
+        pregrant_files=random.sample(AVAILABLE_MAPPING_FILES["pregrant"], k=2),
```

Comment:
Do we have only 2 files in pregrant? If so may be it makes sense to just list them here rather than sampling?